tests/transports-integrations/tests/utils/common.py

```
@@ -0,0 +1,416 @@
+                )
+
+    # Handle Anthropic format: response.content with tool_use blocks
+    elif hasattr(response, "content") and isinstance(response.content, list):
+        for content in response.content:
+            if hasattr(content, "type") and content.type == "tool_use":
+                tool_calls.append({"name": content.name, "arguments": content.input})
+
+    return tool_calls
+
+
+def assert_valid_chat_response(response: Any, min_length: int = 1):
```

Ruff (0.11.9)

Missing return type annotation for public function `assert_valid_chat_response`

Add return type annotation: `None`

(ANN201)